Added new command "okta-iam-list-user-applications" (#11698)
demisto/content: Packs/Okta/ReleaseNotes/2_1_7.md

#### Classifiers
##### Okta IAM - App Sync (classifier)

> - Added the classification for Okta log update user profile events to **IAM - App Update** incident type (used for the **Identity Lifecycle Management (ILM)** pack).

Reviewer: ```suggestion
- Added a classification from user profile update events in Okta to the **IAM - App Update** incident type (used in the **Identity Lifecycle Management** pack).
```

#### Integrations
##### Okta IAM

> - Added command: ***okta-iam-list-user-applications***.

Reviewer: ```suggestion
- Added the ***okta-iam-list-user-applications*** command.
```

Author: This is after rn review. Don't think we should change it

#### Classifiers
##### Okta IAM - App Sync (classifier)

> - Added classification for Okta log - update profile event (used for Identity and Lifecycle management).

Reviewer 2: ```suggestion
- Added the classification for Okta log update user profile events to **IAM - App Update** incident type (used for the **Identity Lifecycle Management (ILM)** pack).
```

#### Integrations
##### Okta IAM
- Added new command: *okta-iam-list-user-applications*

> - Added support for update user in fetch query.

Reviewer 2: ```suggestion
- Added support for fetching update user profile events.
```
